i18n/en.json

```diff
@@ -4350,10 +4354,6 @@
     "id": "ent.elasticsearch.delete_channel.error",
     "translation": "Failed to delete the channel"
   },
```

Comment:
ci is failing on this, but running `make i18n-extract` decides to remove this. any ideas what's going on?

Reply:
nevermind, had the enterprise repo a bit out of date. updating and running again allowed to fix